go/ql/test/experimental/CWE-942/CorsMisconfiguration.go

```diff
@@ -120,7 +120,7 @@ func main() {
 			}
 		})
 		http.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
-			// OK-ish: the input origin header is validated agains a whitelist.
+			// OK-ish: the input origin header is validated against a whitelist.
```

Comment:
I've mostly skipped `/test/` directories, but when I first started, this item jumped out at me, so it's thrown in at no extra charge.